### DISCUSSION TITLE
Is there a documentation on what happens/can happen when the phase changes?

### DISCUSSION THREAD

**Discussion by johann-petrak** [codalab/codabench#1401] [General]
I am not sure how phase changes influence the various aspects of a configured competition. 
* will there be a new leaderboard for the phase?
* how will submissions show up differently in the dump so we can tell them apart?
* what will be shown if there is a period of time BETWEEN two phases?

I wanted to test all this but the bugs related to properly configuring the phase start/end datetimes make this almost impossible.

**Comment by Didayolo** [comment]
Hi @johann-petrak,

> will there be a new leaderboard for the phase?

Yes, each phase is associated to an independent leaderboard

> how will submissions show up differently in the dump so we can tell them apart?

Dumps do not include submissions and results. Only what define the competition to upload it again (documentation, programs, etc.)

> what will be shown if there is a period of time BETWEEN two phases?

Messages say that the phases are closed (either already closed or not opened yet), and the leaderboards remain normally accessible.
